test/integration/suites-windows/windows-service/01-start-server-service

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 source ./common
 
-docker-up spire-server
+docker-spire-server-up spire-server
```

Comment:
It seems that the `check-server-started()` check is not working well on Windows.

Reply:
Seems like this docker-compose suite doesn't bring the server up with the container. It  ends up being with the following commands.

I'll just leave this one like this for now, if it ends up being a problem we can revisit. It asserts the windows service is up, so better than nothing, but doesn't actually check that it is ready to serve requests.